pkg/venafi/tpp/sshCertUtils.go

```diff
@@ -241,3 +242,125 @@ func convertToGenericRetrieveResponse(data *certificate.TppSshCertRetrieveRespon
 	return response
 
 }
+
+func getSshConfigUrl(key, value string) string {
+	var url string
+	query := fmt.Sprintf("%s=%s", key, value)
+	query = netUrl.PathEscape(query)
+	url = fmt.Sprintf("%s?%s", urlResourceSshCAPubKey, query)
+	return url
+}
+
+func RetrieveSshConfig(c *Connector, ca *certificate.SshCaTemplateRequest) (*certificate.SshConfig, error) {
+
+	var url string
+	if ca.DN != "" {
+		fullPath := getCaDNFullPath(ca.DN)
+		url = getSshConfigUrl("DN", fullPath)
+		fmt.Println("Retrieving the configured CA public key for template:", fullPath)
+	} else if ca.Guid != "" {
+		url = getSshConfigUrl("guid", ca.Guid)
+		fmt.Println("Retrieving the configured CA public key for GUID:", ca.Guid)
```

Comment:
In all cases, the user specifies CA template. It can be by name or GUID. Maybe, it makes sense to indicate that the GUID is for the template. 
```suggestion
		fmt.Println("Retrieving the configured CA public key for template with GUID:", ca.Guid)
```

Reply:
fixed on the most recent commits